crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -8552,6 +8552,40 @@ a'a'd'a'a
   a'a'a
 ENDMAP
 
+NAME:   hellmonk_guarded_unrand_hermit
+TAGS:   no_monster_gen
+DEPTH:  D:3-8
+WEIGHT: 3
+KFEAT:  _ = altar_ecumenical
+KITEM:  _ = hermit's pendant no_pickup
+NSUBST: E = 1:. / *=xxx., H = 1:. / *=xxx., I = 1:. / *=xxx., J = 1:. / *=xxx.
```

Comment:
Recommend syntax like:
```
NSUBST: E = . / xxx., H = . / xxx., I = . / xxx., J = . / xxx.
```
It's better to use those implicit NSUBST assignments that effectively do `1=` for non-final entries and `*=` for the final entry.

Reply:
Another minor point, `I` is actually taken by the orcish idol feature, so ideally you'd use a different glyph there. We should probably undefine that one as a default since it no longer has any special function.